example/reverse-proxy/etc/service.conf

```diff
@@ -0,0 +1,40 @@
+map $server_port $int_hostname {
+    "443" "service1";
+
+    "8443" "service2";
+
+    default "localhost";
+}
+
+map $server_port $int_port {
```

Comment:
Well, I am dealing with WebRTC for video chatting, and there are tons of ports needed to communicate with all the necessary services from web browser. And all of them must go through SSL to avoid the warnings obviously. Mapping them by location or subdomain is certainly possible, but it's arguably harder to maintain. Location - possible future name collision between the site path and mapped service. Subdomain - need several nginx-le instances and some way to distinct them.

Reply:
Sorry for interruption, but maybe it will be better to provide simpler service.conf with just multiple `server` blocks and separate "advanced" service configuration with these maps and description what it's preferable for setup which is utilizing utilizing WebRTC.
Providing such a complex setup as an example without explanations why it's done that way and not the way nginx documentation suggest from my POV is harmful for whoever will use it.

Reply:
@paskal agree. Tried to make it clear in the readme, take a look please.

Reply:
The code (example/webrtc/etc/service.conf) have not a single comment.  I'd expect to see maps and proxy part documented, from current code it's not clean why do you need maps instead of multiply server\location blocks, and how it's related to webRTC.

Reply:
Well we cant have comments in nginx conf unfortunately.

Reply:
I will try to replicate this in readme.

Reply:
Oh, it actually does accept `#`. Will use it.